README.md

```diff
@@ -1 +1,68 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO_MICROVERSE
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
```

Comment:
- [ ] The screenshot link leads to 404 page. Kindly fix the issue I think it will make your Readme professional.